tests/rbd/rbd_utils.py

```diff
@@ -75,6 +76,26 @@ def random_string(self):
         temp_str = "".join([random.choice(string.ascii_letters) for _ in range(10)])
         return temp_str
 
+    def initial_rbd_config(self, rbd, pool, image, size, **kw):
+        """
+        Calls create_pool function on the clusters,
+        creates an image in the pool,
+        Args:
+            **kw:
+            pool - name for pool to be created on cluster
+                # create pool does not catch exceptions, it returns true/false.
+                # so we are returning 1 instead of raising exception
```

Comment:
These comments are not making sense to me in doc string where we are explaining how to use the method config.
Please let me know your thoughts.

Reply:
As discuss in the automation peer review meeting. will be removing these lines

Reply:
Modified the comments.